docs/specification/auth/_index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Authentication
+type: docs 
+weight: 50
+---
+
+{{< callout type="warning" >}}
+Auth is **experimental**, and being drafted for release in the next [revision]({{< ref "/specification/revisions" >}}) of the protocol.
+
+The additions to the base protocol are backwards compatible to revision 2024-11-05; however, **the auth specification may change in backwards incompatible ways** until the next protocol revision.
+{{< /callout >}}
+
+The Model Context Protocol (MCP) provides a standardized way for clients and servers to establish secure communication channels. The authentication framework supports multiple authentication methods to accommodate different security requirements and use cases.
+
+## User Interaction Model
+
+Authentication in MCP is designed to be flexible and secure, supporting various authentication flows that may require different levels of user interaction:
+
+- OAuth 2.0 for standardized authorization flows
+- Credential-based authentication for API keys and similar secrets
+
+Other authentication schemes may also be implemented by clients and servers as extensions to the base protocol.
+
+## Capabilities
+
+Clients that support authentication **MUST** declare their supported authentication methods during [initialization]({{< ref "/specification/basic/lifecycle#initialization" >}}):
```

Comment:
I have to say I don't fully understand why the client needs to indicate supported authentication. Does the client and server need to negotiate on the specific capability? In that case, sending the capability from the client side is sensible. However it feels to me that this is not the case here. If there is no negotiation, it is likely enough for the server to send supported authentication and hints if they are required or not and clients disconnect if they don't support it